index.js

```diff
@@ -1,7 +1,73 @@
 // Iteration 1: Names and Input
 
+let hacker1 = "Lucas";
+console.log("driver's name: " + hacker1);
+let hacker2 = "Juan";
+console.log("navigator's name: " + hacker2);
 
 // Iteration 2: Conditionals
+  function which() {
+  if (hacker1.length > hacker2.length) {
+    console.log(`The driver has the longest name, it has ${hacker1.length} characters.`);
+  } else if (hacker1.length < hacker2.length) {
+    console.log(`It seems that the navigator has the longest name, it has ${hacker2.length} characters.`);
+  } else if (hacker1.length === hacker2.length) {
+    console.log(`Wow, you both have equally long names, ${hacker2.length} characters!`);
+  }
+}
+which();
+// Iteration 3: Loops
+//3.1
+const NAME = hacker1.toUpperCase();
+let NAMEnew = "";
+;
+for(let i = 0; i < NAME.length; i++){
+ NAMEnew += NAME[i];
+  if(i !== NAME.length - 1) {
+    NAMEnew += " ";
+  }
+}
+console.log(NAMEnew);
```

Comment:
Super bien!!! te lo voy a limar un poco pero vamos esta perfect!

```suggestion
let newName = '';
for(let i = 0; i < hacker1.length; i++){
 newName += hacker1[i];
}
console.log(newName.trim().toUpperCase());
```

Te incluyo el metodo trim de la coleccion de los string, basicamente lo que hace este metodo es quitarte los espacios por delante y por detras :D, y una cosa buena que tienen los metodos es que se pueden ir concatenando, si quieres escribeme por slack y indagamos en esto! :D